invitation.txt

```diff
@@ -0,0 +1,86 @@
+for use by your maintainers.
+We'd especially like to give them to any of your maintainers
+who aren't already using any.
+This is part of the "The Great MFA Distribution Project"
+<https://github.com/ossf/great-mfa-project> for
+developers of open source software (OSS)/Free Software to improve
+confidence in authentication and integrity of code being delivered.
+
+By **2021-12-20** and preferably much sooner,
+let us know how many you'd like.
+We can provide your project up to 5 Titan tokens from Google and up to
+5 Yubikey tokens from GitHub.  If you could use more, please let us know that
+too. We know many projects could use more, and we may
+be able to provide more at some point.
+In your response, tell us an email
+address where we can *privately* send you the coupon codes and
+validation codes needed to get the tokens.
+
+We would send you coupon codes and validation codes to the private email
+address.
+You would send then distribute those codes to the maintainers you choose.
+The recipients would use the coupon codes and validation codes to "buy"
+the tokens from the Google Store and/or GitHub Shop, who would ship
+the tokens directly to recipients.  
+These codes are use-once, so
+make sure you can keep the codes private until they're used by the
+intended person.
+
+**Important**: The Google coupon codes **must be used by 2021-12-31** on
+the Google Store or they expire.
+
+To qualify, each token recipient must:
+
+1. Be a maintainer or contributor to this critical open source software (OSS)
+   project, or to another OSS project that this project depends on
+   (the dependency may be indirect).
+2. Try to use an MFA token once they receive the token.
+   We'd like recipients to use MFA tokens from then on, but at least try.
+3. Not reuse the token between different people (the token must not be shared).
+4. Consider providing feedback to us (so we can try to fix problems).
+
+We also need each project that receives coupon codes and/or validation codes
+to tell us these numbers (preferably within 30 days of getting the codes):
+
+1. How many tokens did you distribute from just Google? From just GitHub?
+2. How many people received tokens from just Google? From just GitHub? From both?
+3. How many people didn’t have hardware tokens they used for OSS who received tokens from just Google? From just GitHub? From both?
+
+We ask for this information so we can tell others some simple
+measures of success. We don't need nor want the names of any
+individuals participating.
+
+Please note that the tokens are shipped from the US, so while they
+can be shipped internationally, we can't ship somewhere if that is
+forbidden (sanctioned) under US law
+So at this time we are unable to ship to individuals in China,
+Afghanistan, Russia, Ukraine, North Korea, Iran, Sudan, and Syria.
+Sorry about that.
+More information is here: <https://home.treasury.gov/policy-issues/financial-sanctions/sanctions-programs-and-country-information>
```

Comment:
@david-a-wheeler I'm wondering what exactly prevents the GitHub store from shipping Yubikeys to Russia for example? Why would they allow me to fill out the form like that:

![Screenshot 2021-12-10 at 11 33 26](https://user-images.githubusercontent.com/10195800/145542939-2b8c6501-5cdd-4fff-92f5-c1602bc9d1a7.png)

Reply:
@david-a-wheeler  Just to be sure I went to the official Yubico store on Amazon US and they can ship Yubikeys to Russia as well
![Screenshot 2021-12-10 at 21 30 42](https://user-images.githubusercontent.com/10195800/145624638-8321b38e-c95b-4ffa-9ddd-474c097c99ac.png)

so it seems to me that at least two whole countries on the list shouldn't have been "banned" by OSSF because the sanctions are actually imposed against certain regions and individuals. Anyway, even if those countries are really "banned", the link to https://home.treasury.gov/policy-issues/financial-sanctions/sanctions-programs-and-country-information isn't exactly helpful